docs/Database.md

```diff
@@ -0,0 +1,30 @@
+# Minivirt database
+
+Minivirt stores images and VMs in a database, which is just a bunch of folders and files. By default it's located at `~/.cache/minivirt`, but it can be changed by setting `$MINIVIRT_DB_PATH`.
+
+## Images
+
+An image is an immutable snapshot of a virtual machine. It typically consists of a qcow2 disk image and a configuration file. It's identified by an `id` which is calculated based on its files using SHA256. The image files reside in a folder named `{db}/images/{id}`.
+
+### Tags
+
+An image may have tags, which are named references to the image. On disk they are implemented as symlinks to the image directory and are located at `{db}/images/{tag}`.
+
+## VMs
+
+A VM is an instance of a virtual machine. Each VM has a unique `name` and its files reside in a directory like `{db}/vms/{name}`.
```

Comment:
One `VM` being an instance of a `virtual machine` sounds... recursive. Maybe you meant to write `image`, but on the other hand you wrote below that `VM`s are _usually_ derived from `image`s.

Reply:
Right, the intention was to define the term _VM_ in the context of Minivirt. I'll make it italic.